data/membership-schema/src/main/resources/net/corda/schema/membership/member/dynamic/registration/1.0/corda.member.dynamic.registration.json

```diff
@@ -67,6 +67,12 @@
       "description": "Optional. A UUID string that uniquely identifies the pre-auth token.",
       "type": "string",
       "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89aAbB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$"
+    },
+    "corda.serial": {
+      "description": "Optional. The serial on the member's latest member info.",
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 10000000
```

Comment:
Do we need to enforce an explicit max value if it's an integer now?

Reply:
Good point. I'll remove it.